build.gradle

```diff
@@ -181,6 +181,7 @@ allprojects {
 
     // Support different IDEs
     apply plugin: 'idea'
+    idea.module.inheritOutputDirs = true
```

Comment:
This is not relevant to the change itself, but for running tests from Idea. I wasn't able to run tests in idea on h2o-scala subproject due to conflicting output paths in idea between several projects. This fixes it.

Reply:
Perfect @jakubhava, thank you!